docs/src/release-notes-js.md

```diff
@@ -5,6 +5,73 @@ title: "Release notes"
 
 <!-- TOC -->
 
+## Version 1.22
+
+### Highlights
+
+- Role selectors that allow selecting elements by their [ARIA role](https://www.w3.org/TR/wai-aria-1.2/#roles), [ARIA attributes](https://www.w3.org/TR/wai-aria-1.2/#aria-attributes) and [accessible name](https://w3c.github.io/accname/#dfn-accessible-name).
+
+  ```js
+  // Click a button with accessible name "log in"
+  await page.click('role=button[name="log in"]')
+  ```
+
+  Read more in [our documentation](./selectors#role-selector).
+
+- New [`method: Locator.filter`] API to filter an existing locator.
+
+  ```js
+  const buttons = page.locator('role=button');
+  // ...
+  const submitButton = buttons.filter({ hasText: 'Submit' });
+  await submitButton.click();
+  ```
+
+- New web-first assertion [`method: PageAssertions.toHaveScreenshot`] that
+  waits for screenshot stabilization and enhances test reliability.
+
+  The new assertion has screenshot-specific defaults, such as:
+  * disables animations
+  * uses CSS scale option
+
+  ```js
+  await page.goto('https://playwright.dev');
+  await expect(page).toHaveScreenshot();
+  ```
+
+  The new [`method: PageAssertions.toHaveScreenshot`] saves screenshots at the same
+  location as [`method: ScreenshotAssertions.toMatchSnapshot`].
+
+- (preview) Components Testing
```

Comment:
Let's put "(preview)" at the end like in our presentation and as a first item of the release notes.